proposals/0723-qualified-strings.rst

```diff
@@ -0,0 +1,282 @@
+Qualified Strings
+=================
+
+.. author:: Brandon Chinn
+.. date-accepted:: Leave blank. This will be filled in when the proposal is accepted.
+.. ticket-url:: Leave blank. This will eventually be filled with the
+                ticket URL which will track the progress of the
+                implementation of the feature.
+.. implemented:: Leave blank. This will be filled in with the first GHC version which
+                 implements the described feature.
+.. highlight:: haskell
+.. header:: This proposal is `discussed at this pull request <https://github.com/ghc-proposals/ghc-proposals/pull/723>`_.
+.. sectnum::
+.. contents::
+
+This proposal proposes extending ``-XQualifiedDo`` to literal strings, to enable more ergonomic and more powerful syntax than ``OverloadedStrings``. Another way to view this proposal would be extending ``-XRebindableSyntax`` to literal strings, but only within a local scope.
+
+See also:
+
+* `QualifiedLists <https://github.com/ghc-proposals/ghc-proposals/pull/724>`_
+* `QualifiedNumerics <https://github.com/ghc-proposals/ghc-proposals/pull/725>`_
+
+Motivation
+----------
+
+Problems with Type Class-driven overloading
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+``OverloadedStrings`` works by desugaring to a type-class-overloaded function. That leads to certain general shortcomings:
+
+* It is a module-wide setting
+
+  * Anecdotally, people would rather avoid ``OverloadedStrings`` than deal with overloaded lists/strings in the entire module.
```

Comment:
since the proposals were split I think this doesn't need to reference lists anymore

Reply:
Yes, missed this one, thanks!